### DISCUSSION TITLE
browser extension?

### DISCUSSION THREAD

**Discussion by mayank99** [projectwallace/css-analyzer#342] [Ideas]
Quite often I need to analyze styles *locally* on a website that isn't published yet. Would it be possible to make css-analyzer work inside a browser extension? This would enable a faster feedback loop, catching issues before they even get pushed.

**Comment by bartveneman** [comment]
A while ago I already worked on an extension, Firefox only for now: https://addons.mozilla.org/en-GB/firefox/addon/css-analyzer/

It's very early stages and pretty rough. It also doesn't share any code with projectwallace.com, I just copied over all the code just to get started. Could you maybe give this a try and let me know if you run into any glaring issues? 

I know there's some issues with CORS headers in external CSS files for some sites, but maybe it'll do the job for you.
